content/workshops/rhosp_101/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Red Hat OpenStack Platform 101
+menu:
+  main:
+    parent: workshops
+---
+
+# Introduction
+
+This workshop on Red Hat OpenStack Platform is designed as an introduction for system administrators who are intending to implement a cloud computing environment using OpenStack. Students will learn a small taste of how to operate Red Hat OpenStack Platform. In this course will use both Horizon and the unified command-line interface.
+
+# Workshop Login Info
+
+Each student will be assigned a number at the beginning of the workshop. This number will be used throughout the labs. Anywhere you see studentX, substitute the number you were assigned for the X.
```

Comment:
Personal preference (no obligation to do this).

> Anywhere you see studentX, substitute the number you were assigned for the X.

I would put this in an alert box to make it stand out a bit more.